src/components/ProjectionChart.vue

```diff
@@ -68,15 +84,15 @@ export default {
 
       return array[array.length - numberTwo];
     },
-    genData(dateLabels) {
+    buildCurrentDataset(dateLabels, finalDateLabels) {
       const currentDataset = buildChartDataSet(
         this.filteredActivities,
-        dateLabels,
+        this.dateLabels,
```

Comment:
No entiendo, por qué el this?

Reply:
porque lo ocupo en otras partes, entonces preferí dejarlo en las variables del componente